src/Behat/Config/Config.php

```diff
@@ -5,14 +5,27 @@
 namespace Behat\Config;
 
 use Behat\Testwork\ServiceContainer\Exception\ConfigurationLoadingException;
+use PhpParser\BuilderFactory;
+use PhpParser\Node;
+use PhpParser\Node\Name\FullyQualified;
 
 use function is_string;
 
-final class Config implements ConfigInterface
+final class Config implements ConfigInterface, ConfigConverterInterface
 {
+    public const IMPORTS_SETTING = 'imports';
```

Comment:
I have converted several string literals into named constants to make the code more meaningful